test/corpus/arguments.txt

```diff
@@ -133,12 +133,12 @@ NOT an argument
 ================================================================================
 a '{' '}' block
 {foo "{bar}" `{baz}` |{baz| } {}
-foo { bar 
+foo { bar
 { {} foo{{ foo{{{
- {{ 
- {{{ 
+ {{
+ {{{
 { }	foo
-,	inside { }: 
+,	inside { }:
```

Comment:
why was the trailing whitespace removed from the test case?

Reply:
`tree-sitter test --update` did it.

Reply:
I don't remember. I suppose that trailing whitespace is handled by TS implicitly, so probably not important here.

Reply:
I'll keep that in mind then.